node-gui/src/widgets/wallet_mnemonic.rs

```diff
@@ -96,15 +104,32 @@ impl<Message> Component<Message, Theme, iced::Renderer> for WalletMnemonicDialog
                 Text::new(action_text),
                 iced::widget::column![text_input("Mnemonic", &mnemonic).padding(15)],
             )
-            .foot(container(text("Loading...")).width(Length::Fill).center_x())
+            .foot(container(text("Loading...")).center_x(Length::Fill))
         } else {
-            Card::new(
-                Text::new(action_text),
-                iced::widget::column![text_input("Mnemonic", &mnemonic)
+            let body: Element<_> = if self.generated_mnemonic_opt.is_none() {
```

Comment:
Let's rewrite this whole bunch of `if`s as:
```text
let mut element = if self.generated_mnemonic_opt.is_none() {
    use text input element
} else {
    use the non-editable text
};

if state.importing {
    add `text("Loading...")` as a footer
} else {
    add the button as a footer
}
```
so that the dialog remains the same when `state.importing` becomes true